features/400-changing-ptiurn/pnc-data.before.xml

```diff
@@ -0,0 +1,20 @@
+<CPE02>
+  <NMS>
+    <PIS>
+      <PI1 PNCID="XXXX/XX" Filename="NEWURN/JOHN" Trans-Filename="11111948MU0" CRO-NO="" DOB="0" SEX="" COLR="" Height="" PLACE-OF-BIRTH="000000000" ETHNIC-APPEARANCE="" DRIVER-NMBR="" NATIONALITY1="" NATIONALITY2="" NATIONALITY3="" BUILD="" HAIR="" HAIR-TYPE="" HAIR-TEXT1="" HAIR-TEXT2="0" EYE-COLOUR1="0" EYE-COLOUR2="" FACE-HAIR="O MALE UNKNOWN UNKNOWN" SHOE-SIZE-MTRC="" GLASSES="" HANDED="" ACCENT="" BRC-STATUS="" BRC-SOURCE-CONS="" BRC-SOURCE-FULL="" SEX-LIT="" COLR-LIT="" ETHNIC-APP-LIT="" HGT-LIT="" BUILD-LIT="" SHOE-SIZE-LIT="" GLASSES-LIT="" HANDED-LIT="" HAIR-LIT="" EYE-COLOUR-LIT="" ACCENT-LIT="" NATIONALITY-LIT1="" NATIONALITY-LIT2="" NATIONALITY-LIT3="" FACE-HAIR-LIT1="" FACE-HAIR-LIT2="" FACE-HAIR-LIT3=""/>
+    </PIS>
+    <IPS>
+      <IP1 CATEGORY-CODE="1" DESCRIPTION="Impending Proscution" COUNT="NaN" OFF-COUNT="1" FIRST-DATE="" LAST-DATE="" PERSISTENT-YOUNG-OFFENDER-MARKER="" PERSISTENT-OFFENDER-MARKER=""/>
+      <IP2 OFF-COD="17" OFF-CNT="1" FIRST-DATE="2011" LAST-DATE="2011" OFF-COD-LIT="MISCELLANEOUS OFFENCES"/>
+      <IP3 OCC-NMBR="001" ARREST-SUMMONS-REF="1101ZD01410788T" PROS-AGNT="CPS" NAME-CHARGED-SER-NO="000" FILENAME-ALIAS-NAME="NEWURN, JOHN" STAGE-OF-PROCESS="" STAGE-OF-PROCESS-DATE="" FORCE-NAME="">
+        <IP4 ARREST-CHARGE-NUMBER="1" COURT-CHARGE-NUMBER="1" ADJUDICATION="" COMMITTED-ON-BAIL="N" OFF-STATUTE="ROAD TRAFFIC ACT 1988 s.143(2)" OFF-DECODE="USING VEHICLE WHILE UNINSURED" OFFENCE-START-DATE="28112010" OFFENCE-END-DATE="" OFF-CDE-4-LEVEL="01" OFF-CDE-ACPO-MAJ="012" OFF-CDE-ACPO-OFF="013" OFF-CDE-ACPO-SUMM="15" OFF-CODE-CJS="RT88191" OFF-QUAL="" OFF-TIC="0000" PLEA="">
+          <IP5 COURT-CASE-REF="XX/XXXX/XX" COURT-SENTENCE-DATE="26092011" COURT-CODE="2732" COURT-NAME="CROYDON" COURT-TYPE="M" COURT-TYPE-LIT="MAGISTRATES" REASON-FOR-VARIATION="" SUBS-APPR-LIT=""/>
+        </IP4>
+      </IP3>
+    </IPS>
+    <CVS/>
+    <NCS/>
+    <NGS/>
+    <RBS/>
+  </NMS>
+</CPE02>
```

Comment:
new lines?

Reply:
It's generated by a script to be exactly what the test tool returns so I don't really want to change the formatting